File: Data/liteData.py
import pandas as pd

CSVfilepath = "/Users/zanzver/Documents/BCU2/Masters/CMP7206-A-S1-2022:3_Data_Mining/Coursework/DataMiningCoursework/Data/hotel_bookings.csv"
dataset = pd.read_csv(CSVfilepath)

liteDataset = dataset.head(10000)
CSVLitePath = "/Users/zanzver/Documents/BCU2/Masters/CMP7206-A-S1-2022:3_Data_Mining/Coursework/DataMiningCoursework/Data/hotel_bookings_lite.csv"
liteDataset.to_csv(CSVLitePath)

File: Data/liteData.R
# Original file path
CSVfilepath <- "/Data/hotel_bookings.csv"
# Save CSV to dataframe
wholeData <- read.csv(CSVfilepath,stringsAsFactors = TRUE, header = TRUE, sep = ",")

# Get first 10000 rows as lite dataframe
liteData <- head(wholeData, 10000)
# Declare lite data path
CSVLitePath <- "/Data/hotel_bookings_lite.csv"
# Save lite dataset to the folder
write.csv(liteData, CSVLitePath)